### PR TITLE
Add Fuse

### PR DESCRIPTION
**Pull request by Andrew-Pohl**

### CODE REVIEW

File: projects/fuse-FUSE/info.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    {
+      "label": "LP Rewards",
+      "url": "https://rewards.fuse.io/"
+    }
+  ],
+  "socials": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/Fuse_network/",
+      "handle": "Fuse_network/",
```
Comment (fnnhnsn) [inline projects/fuse-FUSE/info.json:29]: Hey, could you just use the Twitter handle without the trailing /, so just `Fuse_network`? Thanks!:)

File: projects/fuse-FUSE/info.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      "search_strings": ["$FUSE", "#Fusenetwork"]
+    },
+    {
+      "name": "Telegram",
+      "url": "https://t.me/fuseio",
+      "handle": "Fuse"
```
Comment (fnnhnsn) [inline projects/fuse-FUSE/info.json:35]: Could you use the trailing url part as Telegram handle, so `fuseio`? Thanks! :)

File: projects/fuse-FUSE/info.json [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    },
+    {
+      "name": "Medium",
+      "url": "https://medium.com/fusenet"
+    },
```
Comment (fnnhnsn) [inline projects/fuse-FUSE/info.json:40]: Could you maybe add a Medium handle, here also the trailing url part, so `"handle": "fusenet"`? Thanks! :)

**Review by fnnhnsn — CHANGES_REQUESTED** [review]
Hey @Andrew-Pohl, thanks a lot for your pr, there are just some minor social network adjustments needed. Thanks in advance for the adjustments!

**Comment by Andrew-Pohl** [comment]
Thanks for the comments @fnnhnsn fixed in 9b50059 :)

**Comment by fnnhnsn** [comment]
Thanks @Andrew-Pohl for the quick fixes! :)

**Comment by fnnhnsn** [comment]
Hey @Andrew-Pohl just one last thing I just ran into concerning the circulating supply: We are starting to add wallets with non circulating token amounts to our list, for example at Tixl [here](https://github.com/tixlfinance/tixlfinance-community-data/blob/master/projects/tixl-txl/info.json), and I was wondering, whether you know if there are such wallets for fuse as well. I just checked [Etherscan](https://etherscan.io/token/0x970b9bb2c0444f5e81e9d0efb84c8ccdcdcaf84d) and found probably the fuse overview page there and wanted to ask whether the tokens of those big holder addresses are yet to be distributed. Then we could add those addresses and generate the circulating supply automatically, which would make the displayed data even more correct in detail. Thanks a lot in advance! :)

**Comment by Andrew-Pohl** [comment]
> Hey @Andrew-Pohl just one last thing I just ran into concerning the circulating supply: We are starting to add wallets with non circulating token amounts to our list, for example at Tixl [here](https://github.com/tixlfinance/tixlfinance-community-data/blob/master/projects/tixl-txl/info.json), and I was wondering, whether you know if there are such wallets for fuse as well. I just checked [Etherscan](https://etherscan.io/token/0x970b9bb2c0444f5e81e9d0efb84c8ccdcdcaf84d) and found probably the fuse overview page there and wanted to ask whether the tokens of those big holder addresses are yet to be distributed. Then we could add those addresses and generate the circulating supply automatically, which would make the displayed data even more correct in detail. Thanks a lot in advance! :)

Hi @fnnhnsn. We have end points to grab circulating and total supply they get updated at the start of each cycle (currently every 48hrs).

Circulating supply (verbose): https://bot.fuse.io/api/v1/stats/circulating
Circulating supply (simple): https://bot.fuse.io/api/v1/stats/circulating_simple
Total supply (verbose): https://bot.fuse.io/api/v1/stats/supply (block number = the fuse block at which the total supply is accurate current inflation = 5% pa)
Total supply (simple): https://bot.fuse.io/api/v1/stats/total_supply_simple

For the list of locked addresses I don't have an endpoint for this (will add one later) the current list is (on both fusenet and eth mainnet):

• 0x805D65Db7F4B532B9dE0c308f8ee41dA25eAc065
• 0x3014ca10b91cb3D0AD85fEf7A3Cb95BCAc9c0f79
• 0xd60ac9e98d5025455E3F68F17AB4a1F962Be5fe7
• 0xd9176e84898a0054680aEc3f7C056b200c3d96C3
• 0x841471619EB0126f48A087bFfBff86b9Df3E1E38
• 0xD418c5d0c4a3D87a6c555B7aA41f13EF87485Ec6
• 0xd617774b9708F79187Dc7F03D3Bdce0a623F6988
• 0x907c0c8642Aa827136A315491dA248071D43B83c

**Comment by fnnhnsn** [comment]
Hey @Andrew-Pohl wow, thanks a lot for all that information! :)

**Comment by Andrew-Pohl** [comment]
> Hey @Andrew-Pohl wow, thanks a lot for all that information! :)

No problem, let me know if you need anything else :)

**Comment by fnnhnsn** [comment]
> No problem, let me know if you need anything else :)

Sure, I'll do that, thanks! :)

**Pull request merged**